src/vue/common/CollectionLoader.spec.js

```diff
@@ -0,0 +1,114 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import CollectionLoader from './CollectionLoader'
+import { globalize } from '../filters/globalize'
```

Comment:
alias

Reply:
you must use aliases everywhere. the only exception you can make is the sibling files (like `./something`)